app/utils/computed-helpers.js

```diff
@@ -16,18 +16,17 @@ export const computedSegmentedLink = function(property) {
       const splitted = this.get(property) ? this.get(property).split('://') : [];
       if (!splitted || splitted.length === 0 || (splitted.length === 1 && splitted[0].includes('http'))) {
         return {
-          protocol : 'https://',
+          protocol : 'https',
           address  : ''
         };
       }
-      const socialUrl = splitted[1].split('/');
       return {
-        protocol : `${splitted[0]}://${socialUrl[0]}/`,
-        address  : socialUrl[1]
```

Comment:
I think the main bug was here, this should happen only for social links and not for other links like `https://api.eventyay.com/`.

Also adding an attribute `isSocialLink` for LinkInput widget.

If this sounds good to you I will create a PR.

Reply:
Go ahead